test/system/users_test.rb

```diff
@@ -4,7 +4,7 @@ class UsersTest < ApplicationSystemTestCase
   setup do
     @emails = ActionMailer::Base.deliveries
     @emails.clear
-    @user = create :user, email: 'superadmin@example.com'
+    @user = create :user, email: 'superadmin@example.com', role: 'superadmin'
```

Comment:
The create :user, as is currently defined, creates by default a superadmin. So unless we need to set another role for the test, the role: is not needed.